server/controllers/expenses.js

```diff
@@ -58,6 +58,10 @@ export const list = (req, res, next) => {
     order: [[req.sorting.key, req.sorting.dir]]
   }, req.pagination);
 
+  if (req.body.unpaid_only || req.query.unpaid_only) {
```

Comment:
I think it's better to keep it consistent with the structure, e.g. req.body.status = 'UNPAID'
And the status should go from 'PENDING' to 'UNPAID' to 'PAID'